crates/ruff_python_ast/src/all.rs

```diff
@@ -39,6 +39,32 @@ impl Ranged for DunderAllName<'_> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct DunderAllNames<'a> {
```

Comment:
Nit: maybe this could be called `DunderAllDefinition` rather than `DunderAllNames`? The name of the struct is very similar to `DunderAllName` currently